packages/fast-components-react-msft/src/select/select.tsx

```diff
@@ -57,9 +57,18 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, {}> {
             <button
                 disabled={props.disabled}
                 className={classNames(select_button)}
-                aria-labelledby={props.labelledBy || null}
-                aria-haspopup={true}
+                id={state.triggerId}
+                role="option"
+                aria-atomic={true}
+                aria-label={state.displayString}
                 aria-expanded={state.isMenuOpen}
+                aria-selected={state.selectedItemIndex !== 0 ? true : false}
```

Comment:
same comment as previous, this conditional evaluates to true/false and since it's used multiple times can be a variable

Reply:
yep